js/my_portfolio.js

```diff
@@ -223,4 +223,49 @@ function render() {
   });
 }
 
+const form = document.querySelector('form');
+
+const errorMessage = document.querySelector('.error');
+
+const email = document.querySelector('#email');
+const emailContainer = document.querySelector('#emailContainer');
+
+function showError(msg) {
+  errorMessage.textContent = msg;
+  errorMessage.classList.remove('d-none', 'd-none-lg');
+  emailContainer.classList.add('error-icon');
+}
+
+function checkemail() {
+  const emailValue = email.value.trim();
+
+  if (emailValue === '') {
+    showError('Email should not be blank');
+    return false;
+  }
+
+  const emailRegex = /[A-Z]/g;
+
+  if (emailValue.match(emailRegex)) {
+    showError(`Email field doesn't allow capital letters. It should be ${emailValue.toLowerCase()}`);
+    return false;
+  }
+
+  return true;
+}
```

Comment:
- [ ] Currently, I entered my email in all uppercase characters and I was able to send an email which is not what is required. It is required to display an error message when a user enters his/her email in uppercase which tells them to enter their email in lowercase characters. Please try fixing this error in other to meet the requirement 😃.

Reply:
Hi @julie-ify, It's working perfectly, we are to generate a function to create an error message when a user submits the form that contains an upperCase character.